### PR TITLE
Rename proxy-container to ipu-storage-container

### PR DESCRIPTION
**Pull request by artek-koltun**
Original proxy-container name was inherited from SPDK docker example.
However, to better convey the idea that it is placed on IPU side, it is better to rename container to highlight that fact.

### CODE REVIEW

**Review by intelfisz — APPROVED** [review]

**Comment by artek-koltun** [comment]
> Original proxy-container name was inherited from SPDK docker example. However, to better convey the idea that this container is placed on IPU side, it is renamed to ipu-storage-container now.

Done

**Comment by mestery** [comment]
When are you all going to work on #158 and #173? We have no GitHub Actions CI for this, and we need it, otherwise we can't verify this works. Further, we need this integrated with the `ipdk` CLI or it doesn't seem like it fits into the broader IPDK repository here.

**Comment by intelfisz** [comment]
> When are you all going to work on #158 and #173? We have no GitHub Actions CI for this, and we need it, otherwise we can't verify this works. Further, we need this integrated with the `ipdk` CLI or it doesn't seem like it fits into the broader IPDK repository here.

This comment does not belong to this PR's conversation. Please add assignees to issues to foster issue awareness and visibility for stakeholders. I will reply in the conversations to the particular issues directly.

File: build/storage/tests/ut/README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Description
-Unit tests covers services written written for IPDK.
+Unit tests covers services written for IPDK.
 
 # Run unit tests
 The command below build all required dependencies and run the unit tests within
```
Comment (intelfisz) [inline build/storage/tests/ut/README.md:5]: builds
runs

File: build/storage/Dockerfile [hunk 2]
```diff
@@ -89,7 +89,7 @@ ARG NO_PROXY
 ################################################################################
 # ipu-storage-container
 #
-# This image is placed on IPU and exposing storage-target NVMe/TCP devices
+# This image is placed on IPU and attaching to storage-target NVMe/TCP devices
```
Comment (intelfisz) [inline build/storage/Dockerfile:92]: missing '.' at the ends of sentences

File: build/storage/tests/ut/README.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # Description
-Unit tests covers services written written for IPDK.
+Unit tests covers services written for IPDK.
```
Comment (intelfisz) [inline build/storage/tests/ut/README.md:2]: covering

**Review by intelfisz — CHANGES_REQUESTED** [review]

File: build/storage/tests/ut/README.md (hunk 1 above)
Comment (artek-koltun) [reply build/storage/tests/ut/README.md]: done

File: build/storage/Dockerfile (hunk 2 above)
Comment (artek-koltun) [reply build/storage/Dockerfile]: done

File: build/storage/tests/ut/README.md (hunk 3 above)
Comment (artek-koltun) [reply build/storage/tests/ut/README.md]: done

**Review by intelfisz — APPROVED** [review]

**Pull request merged**